### PR TITLE
Feature/rnp 146 android side review

### PR DESCRIPTION
**Pull request by PiotrKlis**
We have a codestyle that we should follow in onewelcome projects, you should add it into your IDE and reformat code https://github.com/onewelcome/engineering/blob/master/docs/guidelines/code-styles/OneWelcome.xml

### CODE REVIEW

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt [hunk 1]
```diff
@@ -18,120 +18,95 @@ import javax.inject.Singleton
...
-    }
-
-    interface DeregistrationCallback {
-        fun onSuccess()
-        fun onError(code: String?, message: String?)
-    }
+      })
+  }
+
+  fun getUserProfile(profileId: String?): UserProfile? {
```
Comment (PiotrKlis) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt:65]: When I edited code, I've formatted it with onewelcome code style and that's why there's so many changes in this file.

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/mappers/CustomInfoMapper.kt
Comment (PiotrKlis) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/mappers/CustomInfoMapper.kt:1]: there was a typo `maper` -> `mapper`

**Comment by PiotrKlis** [comment]
There's a compilation problem on bitrise, I must admit I did not compile the code as I do not have react environment setup. It's probably something minor-ish (hopefully) 🤞

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/RegisterUserUseCase.kt [hunk 2]
```diff
@@ -23,6 +23,7 @@ class RegisterUserUseCase @Inject constructor(private val oneginiSDK: OneginiSDK
     operator fun invoke(identityProviderId: String?, scopes: ReadableArray?, promise: Promise) {
         val identityProvider = getIdentityProvider(identityProviderId)
 
+        //Why additional identityProviderId check?
         if (identityProvider == null && identityProviderId != null) {
```
Comment (MichielVrins) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/RegisterUserUseCase.kt:27]: This should probably have a comment above this line. 
It's allowed to pass null as an identityProviderId which would result in the identityProvider also being null, and then we use the default, but when we pass an id that does not exist and we can't find it, then we throw IDENTITY_PROVIDER_NOT_FOUND

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/wrapper/OneginiSdkWrapper.kt [hunk 3]
```diff
@@ -167,23 +167,23 @@ class OneginiSdkWrapper @Inject constructor(
     }
 
     fun registerFingerprintAuthenticator(profileId: String, promise: Promise) {
-        TODO("Not yet implemented")
+        //TODO("Not yet implemented")
```
[the remark this answers is not in the document]
Comment (MichielVrins) [reply android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/wrapper/OneginiSdkWrapper.kt:170]: These are not moved into a usecase yet so their implementation is still in RNOneginiSdk

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/InitializationHandler.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
 package com.onegini.mobile.sdk.reactnative.handlers;
 
 import com.onegini.mobile.sdk.android.handlers.error.OneginiInitializationError;
-
+//Does it has to be java file?
```
Comment (MichielVrins) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/InitializationHandler.java:4]: This file doesn't even need to exist as far as i can see

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/RegisterUserUseCase.kt [hunk 5]
```diff
@@ -20,35 +20,36 @@ import javax.inject.Singleton
...
 
-    operator fun invoke(identityProviderId: String?, scopes: ReadableArray?, promise: Promise) {
-        val identityProvider = getIdentityProvider(identityProviderId)
+  operator fun invoke(identityProviderId: String?, scopes: ReadableArray?, promise: Promise) {
+    val identityProvider = getIdentityProvider(identityProviderId)
 
-        if (identityProvider == null && identityProviderId != null) {
-            promise.rejectWrapperError(IDENTITY_PROVIDER_NOT_FOUND)
-            return
+    //Why additional identityProviderId check?
```
Comment (PiotrKlis) [inline android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/RegisterUserUseCase.kt:26]: Comment is still lingering

File: android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/InitializationHandler.java (hunk 4 above)
Comment (PiotrKlis) [reply android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/InitializationHandler.java:5]: Then burn it with fire 🔥

**Review by MichielVrins — APPROVED** [review]

**Pull request merged**